dbms/src/Processors/Formats/Impl/AvroRowInputFormat.h

```diff
@@ -27,8 +28,9 @@ class AvroDeserializer
 private:
     using DeserializeFn = std::function<void(IColumn & column, avro::Decoder & decoder)>;
     using SkipFn = std::function<void(avro::Decoder & decoder)>;
+    struct SkipFnHolder { SkipFn skip_fn; };
```

Comment:
Don't understand why we need this.
Everything else is Ok.

Reply:
You are right. Got rid of it